src/codeu/chat/DerbyStore.java

```diff
@@ -35,23 +35,24 @@
 	private final String messageTableName = "message";
 	private final String chatParticipantsTableName = "chatParticipants";
 	
-	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getConversationInfo = "SELECT * FROM " + userTableName +  " WHERE id = ?";
-	private final String getUsersInvolved = "SELCT * FROM " + chatParticipantsTableName + " WHERE id = ?" ;
+	private final String getUserInfo = "SELECT * FROM " + userTableName +  " WHERE userid = ?";
+	private final String getConversationInfo = "SELECT * FROM " + conversationTableName +  " WHERE conversationid = ?";
+	private final String getUsersInvolved = "SELECT * FROM " + chatParticipantsTableName + " WHERE conversationid = ?" ;
 	
 	private final String checkUsername = "SELECT * FROM " + userTableName + " WHERE name = ?";
 	private final String checkValidUser = "SELECT * FROM " + userTableName + " WHERE name = ? AND password = ?";
 	
-	private final String addUserInfo = "INSERT INTO " + userTableName + " (id, name, password, creation) values (?, ?, ?, ?)";
-	private final String addConversationInfo = "INSERT INTO " + conversationTableName + " (id, owner, creation, title, firstMessage, lastMessage) " + 
+	private final String addUserInfo = "INSERT INTO " + userTableName + " (userid, name, password, creation) values (?, ?, ?, ?)";
+	private final String addConversationInfo = "INSERT INTO " + conversationTableName + " (conversationid, owner, creation, title, firstMessage, lastMessage) " + 
 			"values (?, ?, ?, ?, ?, ?)";
-	private final String addMessageInfo = "INSERT INTO " + messageTableName + " (id, previous, creation, author, content, nextMessage) " + 
+	private final String addMessageInfo = "INSERT INTO " + messageTableName + " (messageid, previous, creation, author, content, nextMessage) " + 
 	"values (?, ?, ?, ?, ?, ?)";
 	private final String addChatParticipantsInfo = "INSERT INTO " + chatParticipantsTableName + " (conversationid, userid) values (?, ?)";
 
 	
-	private final String updateLastMessageById = "UPDATE " + messageTableName + " SET nextMessage = ? WHERE id = ?";
-	private final String updateConversationById = "UPDATE " + conversationTableName + " SET firstMessage = ?, lastMessage = ? WHERE id = ?";
+	private final String updateNextMessageById = "UPDATE " + messageTableName + " SET nextMessage = ? WHERE messageid = ?";
+	private final String updateConversationById = "UPDATE " + conversationTableName + " SET firstMessage = ?, lastMessage = ? WHERE conversationid = ?";
+	private final String getMessageById = "SELECT * FROM " + messageTableName + " WHERE messageid = ?";
```

Comment:
(commenting here because I can't comment on unchanged lines)

- I'd add some javadoc comments on the public methods and the constructor.
- You're still hardcoding one of the fields in the `addUser()` function.